tests/test_conversations.py

```diff
@@ -0,0 +1,251 @@
+    _setup(client)
+    user_threads_url = reverse('user_conversations:threads')
+    assert user_threads_url == '/conv/user/threads/'
+    user_panel_url = reverse('user_panel:dashboard')
+    assert user_panel_url == '/user-panel/'
+
+    response = client.get(user_panel_url)
+
+    assert user_threads_url in response.content.decode()
+
+
+def test_user_panel_contains_link_to_finaid_request(db, client):
+    _setup(client)
+    new_finaid_url = reverse('user_conversations:new_finaid_request')
+    assert new_finaid_url == '/conv/user/threads/start-new/finaid/'
+    user_panel_url = reverse('user_panel:dashboard')
+    assert user_panel_url == '/user-panel/'
+
+    response = client.get(user_panel_url)
+
+    assert new_finaid_url in response.content.decode()
+
+
+def test_finaid_form_doesnt_work_for_users_that_are_not_logged_in(db, client):
+    new_finaid_url = reverse('user_conversations:new_finaid_request')
+
+    response = client.get(new_finaid_url)
+
+    assert response.status_code == 302
+    assert redirects_to(response, "/accounts/login/")
+
+
+def test_finaid_form_works_for_logged_in_users(db, client):
+    _setup(client)
+    new_finaid_url = reverse("user_conversations:new_finaid_request")
+
+    response = client.get(new_finaid_url)
+
+    assert response.status_code == 200
+    assert template_used(
+        response,
+        "ep19/bs/conversations/user_interface/new_finaid_request.html",
+    )
+
+
+def test_user_can_create_new_finaid_request(db, client):
+    _setup(client)
+    new_finaid_url = reverse("user_conversations:new_finaid_request")
+
+    response = client.post(
+        new_finaid_url,
+        {
+            "full_name": "Joe Doe",
+            "type_of_grant": [
+                UserNewFinaidRequest.FINAID_GRANT_CHOICES.TICKET,
+                UserNewFinaidRequest.FINAID_GRANT_CHOICES.TRAVEL_AND_ACCOMODATION, # NOQA
+            ],
+            "travel_amount": 123,
+            "accommodation_amount": 0,
+            "profession": "IT",
+            "affiliation": "EPS",
+            "country_of_residence": "Poland",
+            "date_of_birth": "2000-01-01",
+            "gender": "Prefer not to disclose",
+            "motivation": "because it's great",
+            "involvement": "github.com/europython/epcon",
+            "expectations": "Learn more and network",
+            "portfolio": "github.com/europython",
+            "how_do_you_use_python": "automation",
+            "first_time_europython": "yes",
+            "speaker_or_coach": "yes",
+            "did_you_volunteer": "yes",
+            "supplements": "My keyboard doesn't do `n` very well :(",
+        },
+    )
+
+    assert response.status_code == 302
+    thread = Thread.objects.get()  # implies there's only one
+    assert thread.category == Thread.CATEGORIES.FINAID
+    assert redirects_to(response, thread.get_user_url())
+
+
+def test_user_panel_doesnt_contain_link_to_helpdesk_inbox_if_user_is_not_staff(
```

Comment:
Marking tests that are todo with `@pytest.mark.xfail` is going to help to keep track of them - otherwise they seem to be working.